storage/operation/instance_params.go

```diff
@@ -0,0 +1,114 @@
+package operation
+
+import (
+	"fmt"
+
+	"github.com/vmihailenco/msgpack/v4"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+const DefaultInstanceParamsVersion = 0
+
+// VersionedInstanceParams wraps instance params with a version tag.
+type VersionedInstanceParams struct {
+	Version        uint64
+	InstanceParams interface{}
+}
```

Comment:
I don't really understand the motivation for making this data structure versioned. `InstanceParams` contains data related to a node's state when it bootstraps, which is immutable after the node has successfully bootstrapped. Because of this, my assumption is that we wouldn't need to change the structure of `InstanceParams` unless the node bootstraps again. But in this case, we are bootstrapping and have a fresh database to work with -- we don't need to worry about different data structure versions...

Versioning data structures is useful but it does have a cost in complexity. In this case, I'm having a hard time finding a use-case that justifies the complexity. 

cc @AlexHentschel

Reply:
> immutable after the node has successfully bootstrapped.
Agree with _existing_ values not being mutated. Though, what is with addition of _new_ fields?

I suspect we will want to extend the instance parameters in the future. In order for this to be rolled out with a protocol upgrade, I was thinking about having it versioned. 

maybe I am overthinking it ... also happy without any versioning

Reply:
Since it's implemented and there is not agreement that we'll definitely _not_ need it, let's keep the versioning then.